jwst/resample/resample.py

```diff
@@ -374,6 +374,29 @@ def update_exposure_times(self, output_model):
         output_model.meta.exposure.end_time = max(exposure_times['end'])
         output_model.meta.resample.product_exposure_time = total_exposure_time
 
+        # Update other exposure time keywords:
+        # XPOSURE (identical to EFFEXPTM)
+        xposure = output_model.meta.exposure.frame_time * (output_model.meta.exposure.ngroups *\
+            output_model.meta.exposure.nframes + (output_model.meta.exposure.ngroups - 1) *\
+            output_model.meta.exposure.groupgap + output_model.meta.exposure.drop_frames1) *\
+            output_model.meta.exposure.nints
+        output_model.meta.exposure.effective_exposure_time = xposure
+        # EFFINTTM (effective integration time)
+        effinttm = (output_model.meta.exposure.ngroups - 1) * output_model.meta.exposure.group_time \
```

Comment:
It looks like this is just computing the value for one model, i.e. one exposure's values of ngroups, group_time, etc. We want the total over all input exposures, so it could be simply computed as the sum of all the `exposure[n].meta.exposure.integration_time` values, as is done in the loop starting at line 366 for the EFFEXPTM values. All of these could/should be computed by just doing that kind of sum over the values from all the input exposures, rather than trying to recompute from scratch.

Reply:
I addressed these changes